### PR TITLE
Janitor Changes

### PR DESCRIPTION
**Pull request by Alex-Selvey**
Changes area above HOP for janitor
Two janitors can spawn round start now
Janidrobe now has double the amount of wardrobe equipment

## Changelog
:cl: MemeProof
del: Removed spawn location in maint janitor room
tweak: Upped amount of goods in JaniDrobe
code: Upped amount of Janitors that can spawn round start to 2
map edit: Revamped the room above HOP to be a Janitor area
/:cl:

jani room change
janidrobe

### CODE REVIEW

**Review by alexkar598 — CHANGES_REQUESTED** [review]
Remove step_y / step_x

**Comment by ghost** [comment]
I prefer the vacant room above HoP. Allows many rp scenarios.

**Comment by Alex-Selvey** [comment]
> Remove step_y / step_x

What exactly do you mean? I was looking through the map dmm and couldn't find a reference for this.

**Comment by Alex-Selvey** [comment]
> I prefer the vacant room above HoP. Allows many rp scenarios.

Dayz had mentioned making a small change to the area and adding an additional roundstart janitor (so now 2 possible round start), we can look at maybe moving it somewhere else.

**Comment by scrubmcnoob** [comment]
The maint is plenty and there's already a ton of vacant areas elsewhere around the station. We do need two janitors though as a cleanly station will be a big role for the upcoming future.

**Comment by Alex-Selvey** [comment]
Should i rollback the map (or just remove the map from the pr) and just add another starting location for a second janitor in the current jani closet?

**Comment by scrubmcnoob** [comment]
No, keep the closet as you have it right now. It looks good.

**Comment by ghost** [comment]
don't remove the dang vacant office

**Comment by SmokingKilz** [comment]
> 
> 
> don't remove the dang vacant office

Vacant office is still there bro? It's right by arrivals.

File: _maps/map_files/NSS-Sunset/NSS-Sunset.dmm [hunk 1]
```diff
@@ -21228,78 +21228,77 @@
...
-/obj/effect/turf_decal/tile/neutral{
-	dir = 1
+/obj/machinery/requests_console{
+	department = "Janitorial";
+	departmentType = 1;
+	pixel_x = 0;
+	pixel_y = 29
+	},
+/obj/item/clothing/gloves/color/orange{
+	step_y = 0
```
Comment (niceping) [inline _maps/map_files/NSS-Sunset/NSS-Sunset.dmm:21297]: step_y

**Review by niceping — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
@SmokingKilz I meant the vacant whatever-that-is. The vacant place-you-can-make-a-shop.

**Comment by SunsetStationuser** [comment]
@MemeProof  this is failing travis

**Comment by Alex-Selvey** [comment]
Probably should just be removed, this was made for metastation and we aren't using that map anymore.

**Pull request closed without merging**